udev/CMakeLists.txt

```
@@ -1,3 +1,7 @@
 install(FILES 50-revpi.rules
   DESTINATION /lib/udev/rules.d
 )
+
+install(PROGRAMS revpi_mac
+  DESTINATION ${CMAKE_INSTALL_LIBDIR}/lib/udev
```

This would result in `/usr/lib/lib/udev`:

```suggestion
  DESTINATION ${CMAKE_INSTALL_LIBDIR}/udev
```

---

It looks like there are really issues with what I edit and what is pushed from my side. This is not what I edited and tested on my end. Same with the old versions that were pushed on friday.